walkthroughs/eks-canary-deployments-pipeline/setup/export_environment_variables.sh

```diff
@@ -0,0 +1,12 @@
+AWS_REGION=us-west-2
```

Comment:
Should this be hard-coded? Customers shouldn't be forced to run these in any given region unless it's against our preview environment.

Reply:
This script was only created to have less code in the blog post content. In the blog post we say that we will be using the region us-west-2 and that the customer can customize these variables if needed. 

I also added a comment to the script:
`# Default variables that were used on the blog post (you can change them to fit your needs)`